fastlane/spec/tools_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe Fastlane do
+  describe "Fastlane::TOOLS" do
+    it "lists all the fastlane tools" do
+      expect(Fastlane::TOOLS.count).to be >= 15
```

Comment:
Should we not verify the actual contents of the array incase a symbol is copy and pasted wrong or twice.

Comparing against the size of the array seems a bit arbitrary to me :)